eox_core/edxapp_wrapper/backends/certificates_h_v1_test.py

```diff
@@ -2,14 +2,14 @@
 Test backend to get GenerateCertificates Model.
 """
 
-from django.db import models
+from django.contrib.auth.models import Permission
 
 
 def get_generated_certificate():
-    """ get test GeneratedCertificate model. """
+    """
+    get test GeneratedCertificate model.
 
-    class CertificatesDummyModel(models.Model):
-        """ Dummy model for testing. """
-        pass
-
-    return CertificatesDummyModel
+    We return any django model that already exists so that
+    django-filters is happy and no migrations are created.
+    """
+    return Permission
```

Comment:
I did not understand why you can pass any model here. I suppose you chose a django built-in one by convenience

Reply:
Exactly. One that already exists. I can pass a new model, but that creates a migration and then tests of the lms fail because `makemigrations` now creates output which the test dislike